src/main/frontend/components/block.cljs

```diff
@@ -1207,7 +1207,8 @@
                             (reset! parents-atom parents)
                             (when (seq parents)
                               (interpose [:span.mx-2.opacity-50 "➤"]
-                                         parents))))]]
+                                         parents))))]
+             component (filterv identity component)]
```

Comment:
Are there any duplications?

Reply:
The `component` data struct is a bit like this  `[div nil [:a ...]]`. These `nil`s will cause a warning.

Reply:
Gotcha